src/pages/about.tsx

```diff
@@ -1,12 +1,71 @@
 import React from "react";
+import { Container } from "../components/Container";
 import { H1 } from "../components/Heading";
 import { Layout } from "../components/Layout";
 
 const About = () => {
   return (
     <Layout>
-      <H1>About The Lemon Press</H1>
-      <p>Page coming soon</p>
+      <Container>
+        <H1 style={{ gridColumn: "1/13" }}>About The Lemon Press</H1>
+        <p style={{ gridColumn: "1/8", display: "flex", alignItems: "center" }}>
+          The Lemon Press is one of only a handful of university satire publications in the U.K.,
+          and has a history of excellence. Nationally recognised, The Lemon Press is one of York's
+          most established and most awarded print media organisations. The print edition is produced
+          once a term, with an additional freshers' edition, and online content is produced on a
+          constant basis. Online, The Lemon Press has a national reach, with videos reaching almost
+          400,000 people. The Lemon Press has recently branched out onto new platforms, including a
+          TikTok and Instagram, and our Multi-Media wing was recently responsible for a Call-To-Arms
+          for supporters of exiled YUSU President Chris Small, which played before a screening of
+          Cats at the student cinema.
+        </p>
+        <div style={{ gridColumn: "8/13" }}>
+          <img
+            src="https://image.freepik.com/free-photo/group-coworkers-office-talking_23-2148985522.jpg"
+            style={{ width: "100%" }}
+          />
+          <p>
+            <i>Pretty much a standard lemon press meeting</i>
+          </p>
+        </div>
+        <div style={{ gridColumn: "1/5" }}>
+          <img
+            src="https://www.publicdomainpictures.net/pictures/220000/nahled/beautiful-bar-in-amsterdam.jpg"
+            style={{ width: "100%" }}
+          />
+          <p>
+            <i>D Bar on a cool winters night</i>
+          </p>
+        </div>
+        <p style={{ gridColumn: "6/13", display: "flex", alignItems: "center" }}>
```

Comment:
Not a massive fan of adding loads of inline styles like this so maybe we can revisit this once we've shown them how it looks and figure out a more reusable solution? Would be nice to maybe style it with Stitches too.